src/clients/db/Repository.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint "@typescript-eslint/camelcase": "off" */
```

Comment:
Can we avoid using comments to disable eslint? I really prefer consistent rules in all files, if some rule doesn't fit for a file maybe it was a wrong rule in the first place?

Reply:
I agree, generally, This is a weird case of why this rule needs to be disabled. 

PouchDB's interfaces have snake_case for some of the options. When those options are used, eslint complains about variables not being camelCase. 

I don't think we should disable this rule across the entire project because our variables should be camelCase. Should we add `.eslintrc.js` in the db folder?

Reply:
Lets leave it like this for the moment. We will see how often it happens.